### PR TITLE
Added kryo-serializer and simple test

### PR DESCRIPTION
**Pull request by alexff91**
Related to issue #610.

### CODE REVIEW

**Comment by rmetzger** [comment]
Hi @alexff91,

thank you for the pull request. I'm sorry that you didn't hear anything from us yet. We are all quite busy right know, but I know from a personal conversation with @StephanEwen that he'll soon take a look over your pull request.

File: stratosphere-java/src/main/java/eu/stratosphere/api/java/typeutils/runtime/KryoSerializer.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+	@Override
+	public int getLength() {
+		return -1;
+	}
+
+	@Override
+	public void serialize(T value, DataOutputView target) throws IOException {
+		this.writer = new Output((OutputStream)target);
+		this.kryoSerializer.register(type);
```
Comment (rmetzger) [inline stratosphere-java/src/main/java/eu/stratosphere/api/java/typeutils/runtime/KryoSerializer.java]: why are you not registering the type in the constructor?

File: stratosphere-java/src/main/java/eu/stratosphere/api/java/typeutils/runtime/KryoSerializer.java [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+import eu.stratosphere.api.common.typeutils.TypeSerializer;
+import eu.stratosphere.core.memory.DataInputView;
+import eu.stratosphere.core.memory.DataOutputView;
+import eu.stratosphere.util.InstantiationUtil;
+
+public class KryoSerializer<T> extends TypeSerializer<T> {
+
+	private static final long serialVersionUID = 1L;
+
+	private final int BUFFER_SIZE = 999999;
```
Comment (rmetzger) [inline stratosphere-java/src/main/java/eu/stratosphere/api/java/typeutils/runtime/KryoSerializer.java]: I think this variable is unused

File: stratosphere-java/src/main/java/eu/stratosphere/api/java/typeutils/runtime/KryoSerializer.java (hunk 1 above)
Comment (alexff91) [inline stratosphere-java/src/main/java/eu/stratosphere/api/java/typeutils/runtime/KryoSerializer.java]: hm,yep, it's better to do in constructor

Comment (rmetzger) [inline stratosphere-java/src/main/java/eu/stratosphere/api/java/typeutils/runtime/KryoSerializer.java]: You probably have to do it in `readObject()` as well

**Comment by StephanEwen** [comment]
I am merging this together with #746 later

**Comment by StephanEwen** [comment]
Merged in 9050111bd30f61c7fed6e545b285c0e945673a82

Thanks for the pull request!

**Pull request closed without merging**